pkg/store/storepb/mock.go

```diff
@@ -0,0 +1,33 @@
+package storepb
+
+import (
+	"context"
+
+	"google.golang.org/grpc"
+)
+
+// StoreClientMock is a structure for mocking StoreClient interface
+type StoreClientMock struct {
```

Comment:
I really think this code gives not much ): How is that helpful than just adding:


```
type mock struct {
storebp.StoreClient
}
func (c *StoreClient) Series(ctx context.Context, in *SeriesRequest, opts ...grpc.CallOption) (Store_SeriesClient, error) {
	// mocked code
}
```

In the test you want mock to have in? I would remove this file and `mock.go` and just add those 5 lines of code in each package you want to mock some logic. What do you think?

Reply:
Okay, totally agree, its excess here. Will remove :)